packages/docs/content/utilities/margin.mdx

```diff
@@ -2,378 +2,234 @@
 title: Margin
 ---
 
-Use the margin utility to change an element's margin.
+Use the margin utility to change the spacing around an item.
+
+## Examples
+
+### Margin on all sides
+
+<UtilityExampleList
+  utilityNames={[
+    '0',
+    '05',
+    '1',
+    '2',
+    '3',
+    '4',
+    '5',
+    '6',
+    '7',
+    'auto',
+  ]}
+   utilityValues={[
+    '0px',
+    '4px',
+    '8px',
+    '16px',
+    '24px',
+    '32px',
+    '40px',
+    '48px',
+    '56px',
+    'auto',
```

Comment:
I can see why we'd want to show that this is a possible value, but the actual example doesn't seem as helpful as it could be. This would kind of ruin the whole genericness of this component and its values, but what if this one's container element had some width on it to show that `--auto` will center it horizontally within its container?

Reply:
This and the fact that we have to duplicate all the spacer information in the mdx content makes me think this component should be specialized to spacing rather than generic. We don't need it to dynamically figure out the pixels, though, since spacing is consistent between themes right now; it can still be simple.

Reply:
Oh, I didn't realize that `PaddingSpacingExample` existed before. If we move back to something like that but refine the design to show the pixel values and expand it to work for margins as well, I think that would work.

Reply:
I hacked together some auto margin ideas last night and am pushing those up to the branch. ATM they are embedded examples.  @pwolfert if you see a way to improve on this or incorporate into the spacing component go for it!